### PR TITLE
CF-1261-updated testing a feed command to include tiamat

### PR DESCRIPTION
**Pull request by stanzikratel**

### CODE REVIEW

**Comment by usnavi** [comment]
ok to test

File: src/docbkx/feeds-publishers-guide.xml [hunk 1]
```diff
@@ -2414,7 +2414,7 @@
                 your feed and events.</para>
             <para>To test your feed, type the following command at a command-line interface:</para>
             <para>
-                <programlisting>mvn test</programlisting>
+                <programlisting> mvn -P tiamat_check test </programlisting>
```
Comment (usnavi) [inline src/docbkx/feeds-publishers-guide.xml]: I might have told you wrong...but its -P tiamat-check :-(   (not an underscore)

**Comment by usnavi** [comment]
FYI - the necessary git commands aren't in this PR.  please check the story & https://one.rackspace.com/display/cloudfeeds/Adding+a+new+feed#Addinganewfeed-XSLTs

File: src/docbkx/feeds-publishers-guide.xml (hunk 1 above)
Comment (stanzikratel) [inline src/docbkx/feeds-publishers-guide.xml]: Just updated the example to say the following:

git submodule update --init --remote cloudfeeds-nabu
mvn -P tiamat-check compile

**Comment by usnavi** [comment]
@stanzikratel - the test phase needs to be run as well:  mvn -P tiamat-check test

Also, please include something along the following:

If errors are discovered, a PR will need to be opened on https://github.com/rackerlabs/cloudfeeds-nabu and the updated XSLT files will need to placed at https://github.com/rackerlabs/cloudfeeds-nabu/tree/master/tiamat/src/main/resources.

After the PR for cloudfeeds-nabu is accepted you can update the submodule and run the check again.
git submodule update --remote cloudfeeds-nabu
mvn -P tiamat-check test

(from https://one.rackspace.com/display/cloudfeeds/Adding+a+new+feed#Addinganewfeed-XSLTs)

File: src/docbkx/feeds-publishers-guide.xml [hunk 2]
```diff
@@ -2410,14 +2410,29 @@
...
-                sample files.</para></section>
+            <para>During the compile phase, the generated XSLT files are verified against the copies
+                in  <link xlink:href="https://github.com/rackerlabs/cloudfeeds-nabu"
+                    >https://github.com/rackerlabs/cloudfeeds-nabu</link>.</para>
+            <para>If any errors are discovered, you need to issue a pull request. Navigate to <link
+                    xlink:href="https://github.com/rackerlabs/cloudfeeds-nabu"
+                    >https://github.com/rackerlabs/cloudfeeds-nabu</link> and copy the updated XSLT
+                files to the <emphasis role="bold"
+                    >https://github.com/rackerlabs/cloudfeeds-nabu/tree/master/tiamat/src/main/resources</emphasis>
+                folder.</para>
```
Comment (usnavi) [inline src/docbkx/feeds-publishers-guide.xml]: @stanzikratel - the term "Navigate to" and "copy the updated XSLT files" aren't completely correct in this context.

I'd say something like:

If any errors are discovered, you'll need to be issue a pull request on https://github.com/rackerlabs/cloudfeeds-nabu.  

After cloning the project,  copy all generated files over to https://github.com/rackerlabs/cloudfeeds-nabu/tree/master/tiamat/src/main/resources, and issue a pull request.

The files to copy are:
- target/xslt-artifacts/rm_private_attrs_for_obs.xsl
- target/xslt-artifacts/nonStringAttrs.xsl
- target/xslt-artifacts/xml2json-feeds.xsl

After the pull request has been accepted.....(and so on, works)

Comment (stanzikratel) [inline src/docbkx/feeds-publishers-guide.xml]: @usnavi Just incorporated the latest changes and pushed them to GitHub

**Pull request merged**